fall18/p5/README.md

```diff
@@ -1,15 +1,15 @@
 # Project 4
 
-This project is for all you movie enthusiasts out there! This week, we will be looking at data set about ~1,000 most popular movies on IMDB in the last 10 years. Each movie in the dataset has like cast, director, rating, genre, runtime, budget, etc.
+This project is for all you movie enthusiasts out there! This week, we will be looking at data set about ~1,000 most popular movies on IMDB in the last 10 years. Each movie in the dataset has information like cast, director, rating, genre, runtime, budget, etc.
 
 We will be working with **strings**, **loops**, **conditionals** and **command line arguments** in this project.
 
 To start, download the 4 files given below into your project directory.
 
-* [main.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/main.py)
-* [project.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/project.py)
-* [test.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/test.py)
-* [IMDB-Movie-Data.csv](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p4/IMDB-Movie-Data.csv)
+* [main.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p5/main.py)
+* [project.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p5/project.py)
+* [test.py](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p5/test.py)
+* [IMDB-Movie-Data.csv](https://raw.githubusercontent.com/tylerharter/cs301-projects/master/fall18/p5/IMDB-Movie-Data.csv)
 
 Open up the IMDB-Movie-Data.csv file and get familiar with it! Notice how the movies are indexed starting from zero, how the 'genres' and 'cast' are listed as several actors separated by spaces. There's something strange about the column 'budget', can you tell what it is?
```

Comment:
I don't see a budget field in the data set. Do you mean the revenue instead?

Reply:
Updated all words budget to revenue